Resources/ArmTemplates/template.json

```diff
@@ -21,12 +21,16 @@
         },
         "environment": {
             "type": "string"
+        },
+        "CosmosDbDatabaseDoesntExist": {
```

Comment:
Shouldn't this be a bool?  Looks like the parameters file has been tokenised to take a bool

Reply:
Updated